include/lmms_math.h

```diff
@@ -25,12 +25,13 @@
 #ifndef LMMS_MATH_H
 #define LMMS_MATH_H
 
+#include <QtGlobal>
+#include <algorithm>
```

Comment:
You add `algorithm` because one of those inline functions here call a function from `algorithm`?

Reply:
I was debugging an undefined reference to `std::clamp` and at some point I put the `algorithm` header in here to see if it would resolve it. Turns out it was because the CMake changes I had at the time werent exporting symbols from `lmmsobjs`, but I left it in here because we should be including what we use anyways.